Hack-A-Sat 2021/Guardians of the/Fiddlin John Carson/README.md

```diff
@@ -0,0 +1,69 @@
+## Fiddlin' John Carson (22 points)
+### Category: "Guardians of the…"
```

Comment:
Can you please fill out the top section the way that we have done other Hack-a-Sat writeups? See `Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef` for an example.

Reply:
@BusesCanFly did you want to change anything that @mossmann suggested?